### ISSUE TITLE
minecraft 1.9

### ISSUE THREAD

**Issue by salamanderrake** [bspkrs/bspkrsCore#28]
Minecraft was updated to 1.9 any chance this and other mods will get updated also?

**Comment by BlueAnanas** [comment]
your whish is fulfilled, however bspkrs still has to merge the PR #29 .
Inbetween you can get the jars from release [Recommended for Minecraft version 1.9](https://github.com/BlueAnanas/bspkrsCore/releases/tag/mc-1.9). I've made releases for the the HUD mods as well.

**Comment by AlvarOsO** [comment]
@BlueAnanas Could you make a release for Treecapitator?

**Comment by salamanderrake** [comment]
yes @BlueAnanas could you make a release for Treecapitator? I think @bspkrs is afk or not interested in these projects anymore.

**Comment by bspkrs** [comment]
Just for the record: I would really like to release updates for my mods for 1.8.9 and 1.9+, and @Lunatrius has even done PRs for 1.8.9 that are 95% done at least, but I just don't have time to even review the code and pull it in right now. Super super busy with a huge business opportunity for the past few months and without a definite end in sight.

So... sorry :(
